client/src/pages/UserPermissionSearch.jsx

```diff
@@ -0,0 +1,288 @@
+    "email": "irissosa@hackforla.org",
+    "projects": ["Home Unite Us", "VRMS Support"]
+  },
+];
+
+const DummyComponent = ({ data, type }) => {
+  return (
+    <List className="search-results disablePadding">
+        {data.map((u, idx) => {
+          // Destructure user object
+          const { _id, name, email } = u;
+          // return projects.length === 0 ?
+          return type === 'admin' ?
+            (
+              <ListItem
+                sx={{
+                  px: 2.4,
+                  py: 1,
+                  borderBottom: 1.6,
+                  borderBottomColor: 'grey.300',
+                }} 
+                key={`result_${_id}/${idx}`}>
+                <ListItemButton
+                  sx={{
+                    px: 0.25,
+                    py: 0.36,
+                    color: 'primary.main',
+                    mx: 0.16,
+                  }}
+                  className="search-results-button"
+                  type="button"
+                  onClick={() => setUserToEdit(u)}
+                >
+                  <Grid container>
+                    <Grid item>
+                      <Typography style={{fontWeight: 600}}>
+                        {`${name.firstName.toUpperCase()} ${name.lastName.toUpperCase()} ( ${email.toUpperCase()} )`}
+                      </Typography>
+                    </Grid>
+                  </Grid>
+                </ListItemButton>
+              </ListItem>
+            ) : 
+              <ListItem
+                style={{display: 'flex', justifyContent: 'flex-end'}}
+                sx={{
+                  px: 2.4,
+                  py: 1,
+                  borderBottom: 1.6,
+                  borderBottomColor: 'grey.300',
+                }} 
+                key={`result_${_id}/${idx}`}>
+                <ListItemButton
+                  sx={{
+                    px: 0.25,
+                    py: 0.36,
+                    color: 'primary.main',
+                    mx: 0.16,
+                  }}
+                  className="search-results-button"
+                  type="button"
+                  onClick={() => setUserToEdit(u)}
+                >
+                  <Grid container justifyContent={"space-between"}>
+                    <Grid item>
+                      <Typography style={{fontWeight: 600}}>{name.firstName.toUpperCase() + " " + name.lastName.toUpperCase()}</Typography>
+                    </Grid>
+                    <Grid item>
+                      <Typography style={{fontWeight: 600}} color='black'>{u.project}</Typography>
+                    </Grid>
+                  </Grid>
+                </ListItemButton>
+              </ListItem>
+            })
+          }
+      </List>
+  )
+}
+
+
+const UserPermissionSearch = ({ users, setUserToEdit }) => {
+  const [userType, setUserType] = useState('admin'); // Which results will diplay
+  const [searchText, setSearchText] = useState(''); // Serch term for the admin/PM search
```

Comment:
There are typos for "display" and "Search" in the comment.

Reply:
Fixed!